dockstore-common/src/main/java/io/dockstore/common/yaml/DockstoreYaml12.java

```diff
@@ -27,12 +29,13 @@
  *
  * Validator requires at least one service or one workflow
  */
-@ValidDockstore12
```

Comment:
seems to have a different implication, in other words I would have assumed that we would create `@ValidDockstore13` if we ever needed to create a new version of the schema. Is the implication that `@HasEntry`  is version agnostic and should just work for all versions?

Reply:
Sure, we can roll the per-version .dockstore.yml-wide annotations into a single parent (such as `ValidDockstore12`), but that'd be one more level of indirection, and I didn't see a need for it, being that we only use the annotation(s) in one place.  `HasEntry` is tied to `DockstoreYaml12`, as the type signature and javadocs indicate, but I can add a version to the name to make sure it's crystal clear.  I should probably edit the comment in the snippet.

Reply:
Added `12` to some class names.